test/unit/styra/systems_test.go

```diff
@@ -0,0 +1,131 @@
+package styra_test
```

Comment:
We usually have unit tests for the styra client in pkg/styra/. So these tests should be added to the existing unit tests found in pkg/styra/systems_test.go